src/controllers/trip.controller.js

```diff
@@ -116,6 +116,24 @@ class TripController {
     ResponseService.setSuccess(200, 'List of all available locations', availableLocations);
     return ResponseService.send(res);
   }
+
+  /**
+ *
+ *
+ * @static
+ * @param {req} req
+ * @param {res} res
+ * @returns {response} @memberof Trips
+ */
+  static async editOpenTripRequest(req, res) {
+    const { userData } = req, userId = userData.id, { requestId, tripId } = req.params;
```

Comment:
One variable declaration per line.

Add this rule to your eslint config file

```json
"one-var": ["error", "never"]
```

Reply:
yes